src/main/resources/pirk.properties

```diff
@@ -28,7 +28,8 @@ local.pirk.properties.dir=/root/
 ##
 ##Spark path for SparkLauncher
 ##
-spark.home = /usr
+#spark.home = /usr
+spark.home=/opt/cloudera/parcels/CDH/lib/spark
```

Comment:
Yes, this is for the CDH cluster that I am doing some of the testing on - I will remove it 

(It is, however, an example of how to alternately set the spark.home property for different cluster configurations)